src/components/buttons/Close/Close.tsx

```diff
@@ -9,10 +9,11 @@ import { ButtonBase } from '../_private/ButtonBase/ButtonBase';
 interface IProps extends IReactComponentProps {
 	position?: 'absolute' | 'static';
 	onClick?: FunctionGeneric;
+	size?: 's' | 'm' | 'l';
 }
 
 const Close: React.FC<IProps> = (props: IProps) => {
-	const { position = 'absolute', onClick, className, ...otherProps } = props;
+	const { position = 'absolute', size = 'l', onClick, className, ...otherProps } = props;
```

Comment:
This isn't something that will make me block this PR, but I'd love to start seeing changes in the props of our components be reflected in the corresponding storybook. So something like what's outlined in this doc:

https://storybook.js.org/docs/react/writing-stories/args

This would help us have a better living documentation.